h3.go

```diff
@@ -287,7 +301,92 @@ func Uncompact(in []H3Index, res int) []H3Index {
 
 // --- REGIONS ---
 
-// TODO(gilley) solve nested c struct problem for Polyfill funcs
+// Convert slice of geocoordinates to an array of C geocoordinates (represented in C-style as a
+// pointer to the first item in the array). The caller must free the returned pointer when
+// finished with it.
+func geoCoordsToC(coords []GeoCoord) *C.GeoCoord {
```

Comment:
can you move these unexported methods to the bottom of the file, with the others?

Reply:
Yep